Setup graphql

jonathandly opened the pull request.

G3Ram commented on howsthewaterfront/src/components/App.js:
```diff
@@ -59,12 +91,15 @@ class App extends React.Component {
+          <Routes childProps={childProps} />
+          {/* Test to make sure apollo is fetching data as expected, will change in future to 
+          display the data appropriately */}
+          <GetBeaches />
```
Daly, Bravo! looks good. I was able to see all the beaches. It will be a good idea to comment out the <GetBeaches /> component. You can use a console.log to print it to the console than print it to the screen.

G3Ram commented:
> Getting an error that there is a conflict with the index file and that it can't resolve apollo/react-hooks so i wasn't able to render the application. Could this because I don't have the .env file? Anyone else having this issue?

@jthicks91 Could be. I have just shared the .env file contents in the main channel. Try adding that and see if it works please.

jonathandly commented:
Yeah I saw when I opened the PR that there is a conflict with index.js
since I didn’t realize that the file was changed last night and I didn’t
have the changes you made in it G3 so I have to add that in then to resolve
the conflict.

On Wed, Sep 4, 2019 at 10:29 PM Gayathri Ramakrishnan <
notifications@github.com> wrote:

> Getting an error that there is a conflict with the index file and that it
> can't resolve apollo/react-hooks so i wasn't able to render the
> application. Could this because I don't have the .env file? Anyone else
> having this issue?
>
> @jthicks91 <https://github.com/jthicks91> Could be. I have just shared
> the .env file contents in the main channel. Try adding that and see if it
> works please.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/howsthewater/frontend/pull/20?email_source=notifications&email_token=AKZYBKZPFSZJFQVKE5YISKLQIBVHPA5CNFSM4ITYLQX2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD55TRVA#issuecomment-528169172>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKZYBK3M7OGBXYLRJ2KOTRDQIBVHPANCNFSM4ITYLQXQ>
> .
>

jonathandly commented:
Yeah Jordan I forgot to keep my imports above the require('dotenv').config(); portion that was causing the error, but I changed it and moved them above that.
